src/Loader.php

```diff
@@ -61,6 +69,11 @@ private function loadXml(SimpleXMLElement $root)
         return $graph;
     }
 
+    /**
+     * @param \SimpleXMLElement $xml
+     * @param \Fhaculty\Graph\Attribute\AttributeAware $target
+     * @param array $keys
+     */
```

Comment:
I add PHPDoc to all methods, regardless of their visibility. It is as much for developers as it is for library users.

Were this a PHP7 with type specified, PHPDoc would have less value.